greenpeace/planet4-plugin-gutenberg-blocks
# PLANET-5151 Migrate EN plugin enform builder, settings to blocks plugin

Migrate the EN plugin functionality from EN plugin repo to blocks plugin

Note: While testing this PR on local you may face a issue of API auth fail.
- Its mainly because the EN API's are IP restricted
- When you update the EN API settings from backend (http://www.planet4.test/wp-admin/admin.php?page=en-settings), the API keys needs to whitelist for your local IP address.
(Feel free to contact me or `Michael Austin` for any help in EN API settings)

Testing instructions -
- Disable the `Planet4 - Gutenberg Engaging Networks` plugin
- Check the EN form builder functionality, it should work similar to EN plugin
